includes/frontend.php

```diff
@@ -436,7 +436,7 @@ public function register_scripts() {
 			[
 				'jquery-ui-position',
 			],
-			'4.9.0',
+			'4.9.3',
```

Comment:
This version was incorrect.

I have updated it, so that we use the same version name in the php and js version of the assets loader.